<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceTile</class>
 <widget class="QWidget" name="SourceTile">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="previewLabel">
     <property name="minimumSize">
      <size><width>320</width><height>180</height></size>
     </property>
     <property name="frameShape"><enum>QFrame::Box</enum></property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
     <property name="text"><string>Preview</string></property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text"><string>Idle</string></property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="timerLabel">
     <property name="text"><string>00:00</string></property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item><widget class="QPushButton" name="settingsButton"><property name="text"><string>Settings</string></property></widget></item>
     <item><widget class="QPushButton" name="startButton"><property name="text"><string>Start</string></property></widget></item>
     <item><widget class="QPushButton" name="pauseButton"><property name="text"><string>Pause</string></property></widget></item>
     <item><widget class="QPushButton" name="stopButton"><property name="text"><string>Stop</string></property></widget></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
